frontend/integration-tests/views/crud.view.ts

```diff
@@ -169,10 +176,12 @@ export const deleteResource = async (resource: string, kind: string, name: strin
 // then navigates back to the original url.
 export const createNamespacedTestResource = async (kindModel, name) => {
   const next = await browser.getCurrentUrl();
-  await browser.get(`${appHost}/k8s/ns/${testName}/${kindModel.plural}/~new`);
+  await browser.get(`${appHost}/k8s/ns/${testName}/${kindModel.plural}`);
+  await isLoaded();
+  await clickCreateWithYAML();
```

Comment:
Why was this change needed?

Reply:
Because it wasn't loaded in FF. I'll test again and come back.

Reply:
> Because it wasn't loaded in FF. I'll test again and come back.

I'm worried that you might have found a bug. You should be able to load these pages directly.

Reply:
Never mind, another "bad flake day" modification, it seems. Dropping...

Reply:
I tested the pink and green code, back and forth, numerous times, and am convinced that FF needs the green code.

Reply:
> I tested the pink and green code, back and forth, numerous times, and am convinced that FF needs the green code.

Again, I'm worried we're changing the test to work around a Firefox bug in console instead of fixing the bug.

Reply:
Fair enough. Given OLM isn't passing in FF, two more places that FF stumbles upon wouldn't make too much of a difference. If I simply revert this, would you accept this PR as "step 1"? We really need these initial enabling changes in place in order to allow broader testing/debugging efforts. Thoughts?

Reply:
I'm OK with that. I'd rather not change this if possible since this test has caught several bugs in PRs before they merged. (We do API discovery and load the OpenAPI schema in the background when console loads, so visiting this page directly is differently than navigating to it from another part of the UI.)

Is it a particularly resource this fails on? It worked OK for me in Firefox visiting the create pod page directly.

Reply:
Thanks to @andrewballantyne , I found a better "fix" for this. It all pointed to "YAML editor page still loading," when pressing that Save button. The `yamlView.isLoaded()` wasn't doing it _enough_ for FF. I've added one more `browser.wait(until.visibilityOf())` call against the "sidebar" that explains the schema for the resource type in the 2 cases where I wanted to change the `/~new` GET calls. I will push once all tests pass locally with both browsers.

Reply:
Oh, wait, it's already been approved/merged? Thanks, I guess, and I'll just open a new PR soon, then. The "debugger experience" with FF has now been verified (at least on Mac), and I'd like a better/easier browser switch (via env var) in next PR (ie. "step 2"). Stay tuned.